### ISSUE TITLE
admin login in QuickConnect broken with jellyfin master

### ISSUE THREAD

**Issue by satmandu** [jellyfin/jellyfin-web#4755]
**Describe The Bug**

https://github.com/jellyfin/jellyfin/pull/8734 and https://github.com/jellyfin/jellyfin-web/pull/4194 merged support for using an admin login with QuickConnect using an admin user to address the feature request in https://github.com/jellyfin/jellyfin-web/issues/3918 . However, using jellyfin-master, it appears this doesn't work.

**Steps To Reproduce**
1. Login to jellyfin (master) using an admin credential
2. Click on hamburger menu, dashboard, Users, username.
3. Click on top right user icon, (hovering over this shows my admin username) 
4. This now takes me to a page with _my admin_ username and a quick Connect option.

**Expected Behavior**
Step 3 & 4 above should take me to a page with the relevant username.

I can manually figure out the UID string for a user, and then construct a URL like https://jellyfinserver/web/#/mypreferencesquickconnect.html?userId=userIdstring

And that DOES work, so the backend is working properly.

**Comment by crobibero** [comment]
This is working as intended, the upper right profile button is always your profile.
See the original issue report for the workflow that was fixed: https://github.com/jellyfin/jellyfin-web/issues/3918


tldr:
To correctly QuickConnect as another user:
1. Go to Dashboard
2. Go to Users
3. Select user to authenticate
4. Click "Edit this user's profile, image and personal preferences."
5. QuickConnect

**Comment by satmandu** [comment]
Thank you! I thought that “Edit this user's profile, image and personal preferences” referred to the page I was on, and didn't realize it was itself a link to a different page.

Might I suggest changing that text to say “Click here to edit this user's profile, image and personal preferences.”
